Sources/ArgumentParser/Usage/DumpHelpGenerator.swift

```diff
@@ -78,7 +78,7 @@ fileprivate extension ArgumentSet {
         argument.help.valueName = group.map(\.valueName).first { !$0.isEmpty } ?? ""
         argument.help.defaultValue = group.compactMap(\.help.defaultValue).first
         argument.help.abstract = group.map(\.help.abstract).first { !$0.isEmpty } ?? ""
-        argument.help.discussion = group.map(\.help.discussion).first { !$0.isEmpty } ?? ""
+        argument.help.discussion = group.map(\.help.discussion).first { !$0.isEmpty } ?? .staticString("")
```

Comment:
thought: I wonder with these empty static strings if it might be worth having either an .empty in the Discussion enum or make these into optional types.

Reply:
I think making Discussion optional is the way to go! Fixed :)